client/assets/javascripts/actions/CommentActionCreators.js

```diff
@@ -58,9 +57,8 @@ export function submitComment(comment) {
   return dispatch => {
     dispatch(incrementAjaxCounter());
     return CommentsManager.submitComment(comment)
-      .then(
-        _comment => dispatch(submitCommentSuccess(_comment)),
-        error => dispatch(submitCommentFailure(error)))
+      .then(_comment => dispatch(submitCommentSuccess(_comment)))
```

Comment:
I should put a note on why _comment. It's because it's in the outer one.

Typically, we use the _something to mean an unused param, but that is not the case here.

We could change the name to

`commentBackFromServer`

because that's what it is.